proto/frequenz/api/dispatch/v1/dispatch.proto

```diff
@@ -65,6 +65,9 @@ service MicrogridDispatchService {
   // Returns a list of all dispatches
   rpc ListMicrogridDispatches(ListMicrogridDispatchesRequest) returns (ListMicrogridDispatchesResponse);
 
+  // Streaming RPC for receiving dispatch updates for a given microgrid
```

Comment:
Maybe it would be worth clarifying a bit more which kind of updates are received via the stream, I guess these kind of updates?

* New dispatch created -> the new dispatch object
* Existing dispatch updated -> the existing dispatch object with the updated values
* Deleted dispatches? Maybe we need a deleted flag, or a deleted timestamp that it is null if it wasn't deleted? -> the deleted dispatch object as it was before being deleted?

Reply:
Yeah, I was wondering if I should add something like an enum event_type { deleted, updated, created }

Reply:
That could work too. I guess for create/update maybe it's not strictly needed but it might be good to detect inconsistencies (if we receive a create but we already have a dispatch with that ID, we know something went wrong).

Reply:
(ping? It was already updated when you wrote this :) )